src/pages/welcomePageTokenReplace.ts

```diff
@@ -11,25 +11,30 @@ export class WelcomePageTokenReplace {
     }
 
     getInstallCommandBasedOnOS(): InstallCommand {
-        let installCommand = new InstallCommand();
         switch (process.platform) {
             case "win32":
-                installCommand.name = "choco";
-                installCommand.command = "choco install gauge";
-                return installCommand;
+                return new InstallCommand({
+                    name: "choco",
+                    displayText: "choco install gauge",
+                    command: "Start-Process -Verb RunAs \"choco\" -ArgumentList \"install -y gauge\"" +
+                    " -WindowStyle hidden -Wait"
+                });
             case "darwin":
-                installCommand.name = "brew";
-                installCommand.command = "brew install gauge";
-                return installCommand;
+            return new InstallCommand({
+                name: "brew",
+                command: "brew install gauge"
+            });
```

Comment:
displayText is missing.

Reply:
`displayText` is defaulted to `command` [here](https://github.com/getgauge/gauge-vscode/pull/208/files/a4da069af82a5354370be517e5df20171817cfe5#diff-78924764452a04f791ea35982ea61bf8R64)

Reply:
but on mac I see `$ undefined` instead of the command.

Reply:
ok, the latest commit should fix this.